com.unity.netcode.gameobjects/Tests/Runtime/NetworkBehaviourUpdaterTests.cs

```diff
@@ -42,97 +42,112 @@ public override IEnumerator Setup()
         public IEnumerator BehaviourUpdaterAllTests([Values(0, 1, 2)] int nbClients, [Values] bool useHost, [Values(0, 1, 2)] int nbSpawnedObjects,
```

Comment:
This is much easier to review if you hide whitespace changes (the entire "setup" section was put into a try/finally so indented)